android/shopify_buy_plugin/build.gradle

```diff
@@ -40,6 +40,7 @@ dependencies {
     testCompile 'junit:junit:4.12'
     testCompile 'org.mockito:mockito-core:1.10.19'
     testCompile 'org.json:json:20140107'
+    testCompile 'org.robolectric:robolectric:3.4.2'
```

Comment:
Oh, never mind I see what you trying to do with
org.robolectric.android.controller.FragmentController